tasks/main.yml

```diff
@@ -12,3 +12,7 @@
 - name: Jobs creds config
   include: job-creds.yml
   when: jenkins_extra_jenkins_role == 'master'
+
+- name: Github Server config
+  include: github-config.yml
+  when: jenkins_extra_github_config_enabled
```

Comment:
I believe an ensuring that github-config.yml doesn't run on slave even when `jenkins_extra_github_config_enabled` is set to `true` in error will be desirable. 

So I'm think you may want:

```
when: jenkins_extra_github_config_enabled and jenkins_extra_jenkins_role == 'master'
```